java/README.md

```diff
@@ -59,6 +59,19 @@ For any other library, such as OkHttp, you will need to include the correspondin
 from the [instrumentation project](https://github.com/open-telemetry/opentelemetry-java-instrumentation) and
 modify your code to initialize it in your function.
 
+### Configuring Context Propagators
+
+#### If you intend on emitting your traces to AWS X-ray (or have enabled Xray Active Tracing)
+Please use the following environment variable for your lambda:
+`OTEL_PROPAGATORS=tracecontext,baggage,xray-lambda`
+
+#### If you intend on emitting your traces to AWS X-ray
+Please use the following environment variable for your lambda:
+`OTEL_PROPAGATORS=tracecontext,baggage,xray`
```

Comment:
This is the default, so this shouldn't be necessary (unless you want to document it as such). It's beneficial to use xray propagation for other tracing services, not just x-ray since that's the only supported propagation mechanism for AWS managed services.

Reply:
oops, this should instead have read "emitting your traces to another system besides AWS X-Ray"

Reply:
I can indicate that this is the default setting